tests/unit/models/blueprint-test.js

```diff
@@ -32,6 +32,17 @@ var beforeInstallCalled;
 var afterInstallCalled;
 var afterUninstallCalled;
 
+function resetCalled() {
```

Comment:
not sure I agree. I don't want to imply I'm doing more than just resetting the "called" variables.